### ISSUE TITLE
local run error

### ISSUE THREAD

**Issue by Ciarponec** [FlutterFlow/flutterflow-issues#5035]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

tried local run as before nov20 update of FF. keeps showing error after that:

Launching lib\main.dart on 23127PN0CG in debug mode...
Upgrading build.gradle
Running Gradle task 'assembleDebug'...                          
Warning: This version only understands SDK XML versions up to 3 but an SDK XML file of version 4 was encountered. This can happen if you use versions of Android Studio and the command-line tools that were released at different times.
ERROR:C:\Users\ciarp\.gradle\caches\transforms-3\3d979c09ae95ea959e87cfa8d7b64639\transformed\jetified-appcompat-resources-1.7.0-runtime.jar: D8: java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null
ERROR:C:\Users\ciarp\.gradle\caches\transforms-3\9ff965d89e7c18328de06a7de7f097d3\transformed\appcompat-1.7.0-runtime.jar: D8: java.lang.NullPointerException: Cannot invoke "String.length()" because "<parameter1>" is null

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeExtDexDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform appcompat-resources-1.7.0.aar (androidx.appcompat:appcompat-resources:1.7.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=23, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: C:\Users\ciarp\.gradle\caches\transforms-3\3d979c09ae95ea959e87cfa8d7b64639\transformed\jetified-appcompat-resources-1.7.0-runtime.jar.
         > Error while dexing.
   > Failed to transform appcompat-1.7.0.aar (androidx.appcompat:appcompat:1.7.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=23, org.gradle.category=library, org.gradle.dependency.bundling=external, org.gradle.libraryelements=aar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: C:\Users\ciarp\.gradle\caches\transforms-3\9ff965d89e7c18328de06a7de7f097d3\transformed\appcompat-1.7.0-runtime.jar.
         > Error while dexing.
   > Failed to transform appcompat-1.7.0.aar (androidx.appcompat:appcompat:1.7.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=23, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: C:\Users\ciarp\.gradle\caches\transforms-3\9ff965d89e7c18328de06a7de7f097d3\transformed\appcompat-1.7.0-runtime.jar.
         > Error while dexing.
   > Failed to transform appcompat-1.7.0.aar (androidx.appcompat:appcompat:1.7.0) to match attributes {artifactType=android-dex, asm-transformed-variant=NONE, dexing-enable-desugaring=true, dexing-enable-jacoco-instrumentation=false, dexing-is-debuggable=true, dexing-min-sdk=23, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for DexingWithClasspathTransform: C:\Users\ciarp\.gradle\caches\transforms-3\9ff965d89e7c18328de06a7de7f097d3\transformed\appcompat-1.7.0-runtime.jar.
         > Error while dexing.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 27s
Running Gradle task 'assembleDebug'...                            148,6s
Error: Gradle task assembleDebug failed with exit code 1

### Expected Behavior

i expect local run to work as before

### Steps to Reproduce

maybe awesome notifications or some other dependency is the reason because new project local run has no issues.
in my case, try local run in the project,
fail with the error.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITFXys/luItJosRE18/AdvlVmiMwGUALa+dI0tZ+TQ0jF5D4E+0Ids/7YVhtZNyzaHxIB1GWnmUDpuiJuITxKu5ePQWbGIB8z8xIThPPbDqWVbbXDJG7OHwnL/FWGkyj0auw2xV+XdpYck4T202QAOysfz6Cf9qOYwx5e6fDbOY=

### Visual documentation

![image](https://github.com/user-attachments/assets/337d606b-3612-4c84-94f6-f4a65722c027)
![image](https://github.com/user-attachments/assets/5aa74f80-c137-48ec-8c0d-a1cbd34109c5)

### Environment

```markdown
- FlutterFlow version: 5.0.12+ (Nov,20 2024)
- Platform: Pc Desktop
- Browser name and version: Chrome 131.0.6778.108
- Operating system and version affected: Windows 11 Home 23H2 version
```

### Additional Information

i'm using java 17. got android studio koala 2024.1.2 installed. using awesome_notifications: ^0.10.0, google_mlkit_text_recognition: ^0.14.0, geolocator: ^13.0.1

**Comment by Alezanello** [comment]
Hello!

From the logs, it appears the issue is related to "parameter3," which often occurs when a component is created but no information is sent to it. However, I couldn't locate this in your project. Have there been any changes to your project since the report?

Is this issue now resolved on your end, or is it still occurring? Please let me know so we can assist further!

**Comment by Ciarponec** [comment]
> Hello!
> 
> From the logs, it appears the issue is related to "parameter3," which often occurs when a component is created but no information is sent to it. However, I couldn't locate this in your project. Have there been any changes to your project since the report?
> 
> Is this issue now resolved on your end, or is it still occurring? Please let me know so we can assist further!

i removed button that i was trying MLkit for 10 mins. then i ctrl-z it back.

**Comment by Alezanello** [comment]
Hello sorry, I didn't understand what you mean with that, in the logs it says that is trying to access to parameter1 on a page or component, did you check where might be?

**Comment by Ciarponec** [comment]
actually the problem is solved by installing Koala specific version : 1.2.1 now i can use local run.
